plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.sqlserver;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+public class SqlServerConfig
+{
+    private boolean snapshotIsolationDisabled;
+
+    public boolean isSnapshotIsolationDisabled()
+    {
+        return snapshotIsolationDisabled;
+    }
+
+    @Config("sqlserver.snapshot-isolation.disabled")
```

Comment:
It is MUCH more common to use `.enabled` in config variables. Consider flipping.

Reply:
Totally. However, note that we have two options here: default mode with automatic selection of snapshot isolation, and a kill switch switch which disables that.
with `sqlserver.snapshot-isolation.enabled=true`  in the config, i would consider connector to use snapshot isolation unconditionally, but this isn't the case.

i could also go with an enum with AUTOMATIC, DISABLED options, but though it's an overkill.

Reply:
Fair enough :)